### PR TITLE
API doc updated

### PR DESCRIPTION
**Pull request by jobarjo**
used code currently on the [apiary](https://app.apiary.io/cosmosexplorer/editor) editor of cosmosexplorer domain.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1,80 +1,138 @@
...
+                        "version": "0.24.0-2ce1abc2",
+                        "channels": "4020212223303800",
+                        "moniker": "ubuntu-xenial",
+                        "other": {
+                            "tx_index": "on",
+                            "rpc_addr": "tcp://0.0.0.0:26657"
+                        }
+                    },
+                    "sync_info": {
+                        "latest_block_hash": "F51538DA498299F4C57AC8162AAFA0254CE08286",
```
Comment (bneiluj) [inline apiary.apib]: we need this: 
```
 "latest_block_hash": "F51538DA498299F4C57AC8162AAFA0254CE08286",
                        "latest_app_hash": "0000000000000000",
                        "latest_block_height": "18",
                        "latest_block_time": "2018-09-17T11:42:19.149920551Z"
```

Comment (bneiluj) [reply apiary.apib]: The Stake Capital API will return clean data to save $ : 
we have endpoint setup: 
```
"network": "game_of_stakes_3",
"version": "0.27.3"
```

and another endpoint status (called more frequently): 
```
      "latest_block_hash": "C3A6C2B4298D6A81A55132AA78AF1D159EE1D276052855B30D77B41837FD24CE",
      "latest_app_hash": "02E672666712B0B234A4F144F04F52A5E4A5C56FC248A54344084279121D7EE4",
      "latest_block_height": "224796",
      "latest_block_time": "2019-01-22T15:02:25.142458104Z",
```

Comment (jobarjo) [reply apiary.apib]: comments taken into account in latest commit.

**Comment by bneiluj** [comment]
@joelb01-github you don't need to close PR. You can make more commits into the same PR. It will automatically update the content.

**Pull request closed without merging**